nf_core/pipeline-template/.github/workflows/cloud_tests_small.yml

```diff
@@ -0,0 +1,76 @@
+name: small-sized tests on cloud providers
+run-name: Submitting workflow to all cloud providers using small sized data
+on:
+  workflow_dispatch:
+    inputs:
+      platform:
+        description: "Platform to run test"
+        required: true
+        default: "all"
+        type: choice
+        options:
+          - all
+          - aws
+          - azure
+          - gcp
+jobs:
+  run-small-tests-on-aws:
+    if: ${{ github.event.inputs.platform == 'all' || github.event.inputs.platform == 'aws' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: seqeralabs/action-tower-launch@v2
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
```

Comment:
We have to add `{%- raw %}` and `{% endraw %}` at the beginning and end of double curly brackets to avoid errors when generating the pipeline from a template. You can see examples on other GHA such as `awstest.yml`

Reply:
Quick summary for @mirpedrol for a follow-up

1. I've minimized the surface area to only cover aws block right now https://github.com/nf-core/tools/pull/2544/commits/0d3aeb235a6d5dca5a3f69b7d1116c902d6e9d6f 

2. The test seems to be passing but there is a warning regarding jinja (still 😞 ) 

<img width="1325" alt="image" src="https://github.com/nf-core/tools/assets/12799326/3f650f95-6c47-4ba4-b155-14e114ed9d9c">

Reply:
I think the if statement on [line 18](https://github.com/nf-core/tools/pull/2544/files#diff-6e7e397d7964d69397b1c7e7240abac5d63186bbffd3e9b6f925d96eadef89e4R18) is the one with curly brackets left, so if you move the comment from [line 20](https://github.com/nf-core/tools/pull/2544/files#diff-6e7e397d7964d69397b1c7e7240abac5d63186bbffd3e9b6f925d96eadef89e4R20) up this should be fixed 🙂  

And I noticed while testing this that the curly brackets in commented out lines need to be skipped too 😮 so the same change should be made for the other jobs.